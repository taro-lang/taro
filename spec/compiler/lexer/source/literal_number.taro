var number : Int = 1
