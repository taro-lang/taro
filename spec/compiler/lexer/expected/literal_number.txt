Var (var) line: 0, length: 3, position: (0,2)
Space ( ) line: 0, length: 1, position: (3,3)
Identifier (number) line: 0, length: 6, position: (4,9)
Space ( ) line: 0, length: 1, position: (10,10)
Colon (:) line: 0, length: 1, position: (11,11)
Space ( ) line: 0, length: 1, position: (12,12)
Identifier (Int) line: 0, length: 3, position: (13,15)
Space ( ) line: 0, length: 1, position: (16,16)
Assign (=) line: 0, length: 1, position: (17,17)
Space ( ) line: 0, length: 1, position: (18,18)
Number (1) line: 0, length: 1, position: (19,19)